app/presenters/supergroups/policy_and_engagement.rb

```diff
@@ -8,6 +8,15 @@ def initialize
 
     def tagged_content(taxon_id)
       @content = MostRecentContent.fetch(content_id: taxon_id, filter_content_purpose_supergroup: @name)
+      @content.select do |content_item|
+        consultation?(content_item.content_store_document_type)
+      end | @content
```

Comment:
`|` is bitwise or in Ruby right?

Reply:
Yeah. I was trying to find a way to get consultations from the list of documents without losing the other documents. There's probably a nicer way to do this.